samples/NovelRT.DotNet.Sample/NovelRT.DotNet.Sample.csproj

```diff
@@ -0,0 +1,34 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net5.0</TargetFramework>
+    <RuntimeIdentifier Condition="'$(RuntimeIdentifier)' == ''">$(NETCoreSdkRuntimeIdentifier)</RuntimeIdentifier>
+    <SelfContained>true</SelfContained>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="../../src/NovelRT.DotNet/NovelRT.DotNet.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
```

Comment:
We should probably just rely on the CMake targets for copying these resources for now; just to be consistent with the rest of the project - and also for making sure CMake knows about any dependencies between the targets.

Reply:
To clarify, this is in case we add any pre-processing steps to our resources which may need special handling (e.g. precompiling shaders to SPIR-V)

Reply:
Relying on CMake doesn't get it to the publish folder, so we need two steps. Likewise, its problematic for anyone trying to use `dotnet build`, which otherwise "works".

We likely need a good setup to ensure that `dotnet build` and `cmake` ultimately do the same thing.

Reply:
Since we're controlling the publish directory from CMake, we can just manually copy to it. It's not the *greatest* solution, and I agree we need a better solution, but that's not going to be fun to implement.